packages/dart_firebase_admin/lib/src/app.dart

```diff
@@ -8,6 +8,7 @@ import 'package:firebaseapis/identitytoolkit/v3.dart' as auth3;
 import 'package:googleapis_auth/auth_io.dart' as auth;
 import 'package:googleapis_auth/googleapis_auth.dart';
 import 'package:meta/meta.dart';
+import 'package:platform/platform.dart';
```

Comment:
I don't like adding a new dependency just for this.

Reply:
One way to do this would be to use zones.

You could do:

```dart
@internal
const envSymbol = #_envSymbol;

...

final env = (Zone.current[envSymbol] ?? Platform.current.environment) as Map<String, String>;
final credential = env['GOOGLE_APPLICATION_CREDENTIALS'];


...

test(..., () {
  runZoned(() => ..., zoneValues: {envSymbol: { 'GOOGLE_APPLICATION_CREDENTIALS': 'foo', );
```

Reply:
TIL. Thanks @rrousselGit